microshift_storage/microshift-storage-overview.adoc

```diff
@@ -0,0 +1,35 @@
+:_content-type: ASSEMBLY
+[id="microshift-storage-overview"]
+= {product-title} storage overview
+include::_attributes/attributes-microshift.adoc[]
+:context: storage-overview-microshift
+
+toc::[]
+
+{product-title} supports multiple types of storage, both for on-premise and cloud providers. You can manage container storage for persistent and non-persistent data in an {product-title} cluster.
```

Comment:
So this renders as "an MicroShift cluster".

I know you can easily fix this here, but you might want to double check your rendered build to make sure that none of your shared modules are doing this. (If they are, you can always conditionalize, though I realize that is kind of annoying :smile_cat: )